### ISSUE TITLE
Unjustified error message "Invalid byte 2 of 4-byte UTF-8 sequence"

### ISSUE THREAD

**Issue by Matt-1** [w3c/epubcheck#1548]
I've encountered this error and created a stripped-down version of the original EPUB that still exhibits the error: [invalidUtf8Sequence.epub](http://devel.springer.de/invalidUtf8Sequence.epub)

For this file, EPUBCheck v5.1.0 reports

```
Validating using EPUB version 3.3 rules.
FATAL(RSC-016): invalidUtf8Sequence.epub/OEBPS/html/Chapter_6.xhtml(88,792): Fatal Error while parsing file: Invalid byte 2 of 4-byte UTF-8 sequence.
[...]
```
I believe this is a false positive. At least I can't find anything wrong with the HTML file.

**Comment by rdeltour** [comment]
Thanks for the report and the test file! I will have a look.
